engine/crates/postgres-types/src/transport/tcp/transaction.rs

```diff
@@ -0,0 +1,24 @@
+use crate::{error::Error, transport::Transport};
+use async_trait::async_trait;
+use futures::stream::BoxStream;
+use serde_json::Value;
+use tokio_postgres::Transaction;
+
+use super::executor;
+
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+impl Transport for Transaction<'_> {
+    fn parameterized_query<'a>(&'a self, query: &'a str, params: Vec<Value>) -> BoxStream<'a, Result<Value, Error>> {
+        executor::query(self, query, params)
+    }
+
+    async fn parameterized_execute(&self, query: &str, params: Vec<Value>) -> crate::Result<i64> {
+        executor::execute(self, query, params).await
+    }
+
+    // we don't care about this in a tx
+    fn connection_string(&self) -> &str {
+        ""
```

Comment:
maybe we could panic instead? That should never be reached.

Reply:
This is a stupid thing I use for testing only. I'm probably removing it in the future when I have a bit of time.